projects/netgrif-components-core/src/lib/data-fields/file-list-field/file-list-default-field/abstract-file-list-default-field.component.ts

```diff
@@ -360,4 +372,25 @@ export abstract class AbstractFileListDefaultFieldComponent extends AbstractBase
         }
         return this.cutProperty;
     }
+
+    protected checkTypes(itemType: string) {
```

Comment:
This method can be extracted to be used in more components.

Reply:
done